ui/src/app/job-list/table/table.component.html

```diff
@@ -123,7 +124,12 @@
             </mat-form-field>
           </div>
         </div>
-        <div class="display-field" *ngIf="!isStatusField(df) && !canEdit(df)">{{ getFieldValue(j, df) }}</div>
+        <div class="display-field" *ngIf="isFirstColumn(df)">
+          <a *ngIf="isFirstColumn(df)" class="field-value job-details-button mat-body-1"
+             [routerLink]="[j.id]"
+             [queryParams]="getQueryParams()">{{ getFieldValue(j, df) }}</a>
+        </div>
+        <div class="display-field" *ngIf="!isStatusField(df) && !canEdit(df) && !isFirstColumn(df) && !canFilterBy(df.field)">{{ getFieldValue(j, df) }}</div>
```

Comment:
nit: This is a pretty long block of code for an `*ngIf`, could you create a wrapper function which calls all these?

Reply:
done